policyengine_canada/variables/household/income/individual/other_than_eligible_dividend_income.py

```diff
@@ -0,0 +1,10 @@
+from policyengine_canada.model_api import *
+
+
+class other_than_eligible_dividend_income(Variable):
```

Comment:
as i said non-eligible is the convention, are you seeing otherwise?

Reply:
All the names were changed to "non-eligible" now.